src/app/js/socketManager.js

```diff
@@ -198,7 +198,20 @@ function subscribeSendMessageListener () {
 
             const socket = connection.socket
 
-            socket.emit(newMessage.eventName, ...newMessage.message)
+            const messageList = []
+            const objectConstructor = {}.constructor;
+            if (newMessage.messageIsJsonCollection[state.activeTab] !== undefined) {
+              const isJson = newMessage.messageIsJsonCollection[state.activeTab]
+              const message = newMessage.message.pop()
```

Comment:
This way only the last argument of a message will be sent while the message editor support multiple arguments per message.
We still want the entire message to be sent to the server.

Reply:
Thanks for your help. :simple_smile: